examples/hello-world/ml-to-fl/np/README.md

```diff
@@ -143,3 +143,28 @@ Then we can run it using the NVFlare Simulator:
 ```bash
 nvflare simulator -n 2 -t 2 ./jobs/np_loop_cell_pipe -w np_loop_cell_pipe_workspace
 ```
+
+## Launch once for the whole job and with metrics streaming
+
+Sometimes we want to stream the training progress.
+
+We can use the flare.log method to do that: [./code/train_metrics.py](./code/train_metrics.py)
```

Comment:
I don't see flare.log being used in train_metrics.py.

Reply:
fixed